tutorials/conf.py

```diff
@@ -96,7 +96,12 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'astropy_favicon.ico'
+html_favicon = "_static/astropy_favicon.ico"
+
+# (Optional) Logo. Should be small enough to fit the navbar (ideally 24x24).
```

Comment:
Can you explain this addition? With our current theme, I don't think we need to set this, so I would suggest removing this.

Reply:
Yup, sure... we don't need it. I was playing around with it and omitted to delete it.

Reply:
Removed.